### PR TITLE
added new Salvor repos

### PR DESCRIPTION
**Pull request by salvorio**

### CODE REVIEW

File: data/ecosystems/a/avalanche.toml [hunk 1]
```diff
@@ -116,6 +116,7 @@ sub_ecosystems = [
   "RocketX",
   "Roco Finance",
   "RUGenerous",
+  "Salvor",
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:119]: This ecosystem doesn't exist so you'll probably want to create a toml file for it if you want it to be a subecosystem of Avalanche, and then move the salvorio urls into that new toml file

**Review by eherrerosj — CHANGES_REQUESTED** [review]

File: data/ecosystems/a/avalanche.toml (hunk 1 above)
Comment (salvorio) [reply data/ecosystems/a/avalanche.toml:119]: It's done, thanks!

File: data/ecosystems/a/avalanche.toml [hunk 2]
```diff
@@ -2218,6 +2219,9 @@ url = "https://github.com/rytell/hero-staking"
 [[repo]]
 url = "https://github.com/sabanstev/ToDoList"
 
+[[repo]]
+url = "https://github.com/salvorio/salvor-contracts"
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:2223]: no need to add it here if the ecosystem has a subecosystem that has the repo, it operates like a graph and rolls up all the underlying subecosystem repos to compute the attribution later in our ETL, can remove

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a final comment

**Comment by ashucoder9** [comment]
Hey @salvorio, could you please include the above suggestions so that we can get this PR merged?

**Comment by eherrerosj** [comment]
Hi @salvorio, @ashucoder9, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.

Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

**Comment by jubos** [comment]
Manually merged here: [06f5b6a](https://github.com/electric-capital/crypto-ecosystems/commit/06f5b6ac8fc627bf5a93484cdbcc1af75c9233e5)

**Pull request closed without merging**